build.sbt

```diff
@@ -450,52 +451,130 @@ lazy val scalalib =
     )
     .settings(mavenPublishSettings)
     .settings(
-      assembleScalaLibrary := {
-        import org.eclipse.jgit.api._
-
-        val s      = streams.value
-        val trgDir = target.value / "scalaSources" / scalaVersion.value
-        val scalaRepo = sys.env.getOrElse("SCALANATIVE_SCALAREPO",
-                                          "https://github.com/scala/scala.git")
-
-        if (!trgDir.exists) {
-          s.log.info(
-            s"Fetching Scala source version ${scalaVersion.value} from $scalaRepo")
-
-          // Make parent dirs and stuff
+      // code to fetch scala sources adapted, with gratitude, from
+      // Scala.js Build.scala, lines 1125 to 1233, at the suggestion of
```

Comment:
The references to the lines will be lost when the build of Scala.js changes in the future. You can use a fixed URL instead like this:
https://github.com/scala-js/scala-js/blob/1761f94ee31902b61c579d5cb121117c9dc08295/project/Build.scala#L1125-L1233

Reply:
Understood & agreed. Thank you for the suggested replacement.
I had to break up the URL in the code. Not sure if the way I did it
is acceptable.  I think the blob url is a good marker in time and will
simplify the life of anyone trying to understand this code.